Sources/WalletSdk/MDocHolderBLECentral.swift

```diff
@@ -275,7 +266,7 @@ extension MDocHolderBLECentral: CBPeripheralDelegate {
     func peripheral(_ peripheral: CBPeripheral, didDiscoverServices error: Error?) {
         if (error) != nil {
             self.callback.callback(
-                message: MDocBLECallback.error("Error discovering services: \(error!.localizedDescription)")
+                message: .error(.bleStack("Error discovering services: \(error!.localizedDescription)"))
```

Comment:
Similar to returning `central.state` I think it would be good to return the error directly and then have a case for `discoverServices`, `discoverCharacteristicsFor`, etc.